kbase-extension/static/kbase/js/widgets/vis/plants/kbasePMIBarchart.js

```diff
@@ -226,88 +213,88 @@ define (
                                         }
 
                                         const selected_subsystems = [];
-                                        $.each(
-                                            $pmi.$elem.find('.subsystem-checkbox'),
-                                            (i,c) => {
-                                                if (c.checked) {
-                                                    selected_subsystems.push($(c).val());
-                                                }
+                                        $.each($pmi.$elem.find('.subsystem-checkbox'), (i, c) => {
+                                            if (c.checked) {
+                                                selected_subsystems.push($(c).val());
                                             }
-                                        );
+                                        });
 
                                         $pmi.displaySubsystems(selected_subsystems);
-
                                     })
                             )
                             .append(
-                                $.jqElem('ul')
-                                    .addClass('dropdown-menu')
-                                    .css({ width : '450px', 'padding-left' : '5px', 'text-align' : 'left' })
-                            )
-                        ;
-
-
+                                $.jqElem('ul').addClass('dropdown-menu').css({
+                                    width: '450px',
+                                    'padding-left': '5px',
+                                    'text-align': 'left',
+                                })
```

Comment:
AST change

Reply:
Gets shown as part of the FBA results widget, under the Bar Charts tab. Code looks fine, works fine.